docs/content/any/guides/data_filtering/_index.md

```diff
@@ -0,0 +1,621 @@
+---
+title: "Filter Data (Oso 0.20.0 Beta)"
+weight: 1
+showContentForAnyLanguage: true
```

Comment:
Not sure this should be `true`. It's confusing to read the guide without the examples & then get to the bottom and see the language selector, then click it and see all the examples populate.

Reply:
Yeah, I'm gonna hack around this lol

Reply:
Ok, should be better and not show anything for java/go/rust. Didn't wanna actually break out the examples because this is just gonna get rewritten next week.